src/components/ha-labeled-slider.html

```diff
@@ -8,30 +8,42 @@
       :host {
         display: block;
         padding-bottom: 16px;
+        display: flex;
+        flex-direction: row;
```

Comment:
Remove `display: block`, direction row is default

Reply:
Oh, I missed that 
I started on this a couple of weeks ago, and stopped halfway in-between some edits in a not-pretty state....